src/main/java/billgates/use_cases/bill_import/BillImportRequestModel.java

```diff
@@ -0,0 +1,35 @@
+package billgates.use_cases.bill_import;
+
+import billgates.use_cases.insert_entry.InsertEntryController;
+import billgates.use_cases.insert_entry.InsertEntryInputPort;
+import billgates.use_cases.insert_entry.InsertEntryUseCase;
+
+import java.util.List;
+
+/**
+ * Clean Architecture Layer: Application Business Rules
+ * A request model of the Bill Import use case.
+ * It is used to transfer input data as a whole
+ * from the controller to the use case.
+ *
+ * @author Eva
+ * @see BillImportInputPort
+ * @see BillImportController
+ * @see BillImportUseCase
+ */
+
+public class BillImportRequestModel {
+    private List<List<String>> csvFile;
```

Comment:
Update: I see why it is a List of List of Strings. 

However, to avoid confusion, do we want to change the name to something else? 

For example, maybe naming it "importedData" would be more straight forward?